File: mobile/ios/HiAir/DesignSystem/HiAirColors.swift
import SwiftUI

enum HiAirColors {
    enum Surface {
        static let bg0 = Color(hex: 0x050B16)
        static let bg1 = Color(hex: 0x081221)
        static let bg2 = Color(hex: 0x0B1730)
        static let bg3 = Color(hex: 0x101B37)
        static let glassFill = Color(hex: 0x0B1730)
    }

    enum Text {
        static let primary = Color(hex: 0xFFFFFF)
        static let secondary = Color(hex: 0xE4ECF8)
        static let tertiary = Color(hex: 0xC5D4EC)
    }

    enum Cta {
        static let gradientStart = Color(hex: 0x1AE8FF)
        static let gradientMid = Color(hex: 0x4D8CFF)
        static let gradientEnd = Color(hex: 0xA06AFF)
        static let labelOnGradient = Color(hex: 0xFFFFFF)
    }

    enum Spectrum {
        static let cyan = Color(hex: 0x1AE8FF)
        static let electricBlue = Color(hex: 0x4D8CFF)
        static let violet = Color(hex: 0xA06AFF)
        static let magenta = Color(hex: 0xE05CFF)
    }

    enum Risk {
        static let low = Color(hex: 0x35E6A2)
        static let moderate = Color(hex: 0xFFD447)
        static let high = Color(hex: 0xFF8A3D)
        static let veryHigh = Color(hex: 0xFF4D68)
    }

    enum Feedback {
        static let info = Color(hex: 0x21D7FF)
        static let errorSoft = Color(hex: 0xFF4D68)
    }

    enum Overlay {
        static let subtle = Color.white.opacity(0.12)
        static let medium = Color.white.opacity(0.18)
        static let strong = Color.white.opacity(0.26)
        static let borderSoft = Color(hex: 0x1AE8FF).opacity(0.42)
        static let borderGlass = Color(hex: 0xA06AFF).opacity(0.46)
        static let avatarHighlight = Color.white.opacity(0.88)
        static let avatarFeature = Color.white.opacity(0.90)
        static let specular = Color.white.opacity(0.28)
        static let innerBorder = Color.white.opacity(0.34)
    }

    enum Brand {
        static let orbCyan = Color(hex: 0x1AE8FF)
        static let orbViolet = Color(hex: 0xA06AFF)
        static let orbTeal = Color(hex: 0x35E6A2)
        static let orbMagenta = Color(hex: 0xE05CFF)
    }
}

File: mobile/android/app/src/main/java/com/hiair/ui/design/HiAirColors.kt
package com.hiair.ui.design

import android.graphics.Color

object HiAirColors {
    object Surface {
        val bg0: Int = Color.parseColor("#050B16")
        val bg1: Int = Color.parseColor("#081221")
        val bg2: Int = Color.parseColor("#0B1730")
        val bg3: Int = Color.parseColor("#101B37")
    }

    object Text {
        val primary: Int = Color.parseColor("#FFFFFF")
        val secondary: Int = Color.parseColor("#E4ECF8")
        val tertiary: Int = Color.parseColor("#C5D4EC")
    }

    object Cta {
        val gradientStart: Int = Color.parseColor("#1AE8FF")
        val gradientMid: Int = Color.parseColor("#4D8CFF")
        val gradientEnd: Int = Color.parseColor("#A06AFF")
        val labelOnGradient: Int = Color.parseColor("#FFFFFF")
    }

    object Risk {
        val low: Int = Color.parseColor("#35E6A2")
        val moderate: Int = Color.parseColor("#FFD447")
        val high: Int = Color.parseColor("#FF8A3D")
        val veryHigh: Int = Color.parseColor("#FF4D68")
    }

    object Feedback {
        val info: Int = Color.parseColor("#1AE8FF")
        val errorSoft: Int = Color.parseColor("#FF4D68")
    }

    object Overlay {
        const val subtleAlpha = 0x1F
        const val mediumAlpha = 0x2E
        const val strongAlpha = 0x42
        const val borderSoftAlpha = 0x4D
    }

    object Spectrum {
        val cyan: Int = Color.parseColor("#1AE8FF")
        val electricBlue: Int = Color.parseColor("#4D8CFF")
        val violet: Int = Color.parseColor("#A06AFF")
        val magenta: Int = Color.parseColor("#E05CFF")
    }

    object Brand {
        val orbCyan: Int = Color.parseColor("#1AE8FF")
        val orbTeal: Int = Color.parseColor("#35E6A2")
        val orbViolet: Int = Color.parseColor("#A06AFF")
        val orbMagenta: Int = Color.parseColor("#E05CFF")
        val nightTop: Int = Color.parseColor("#050B16")
        val nightBottom: Int = Color.parseColor("#0B1730")
    }
}
